client/src/ui-components/ClientOrdersComponent/ClientOrderTableRow.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { PopUpForCompleteOrder } from "./PopUpForCompleteOrder";
+import "./ClientOrderTableRow.css";
+
+function ClientOrderTableRow(props) {
+  const [status, setStatus] = useState(props.order.status);
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const handleSubmit = () => {
+    if (status === "PREPARED") {
+      setStatus("DONE");
+    } else if (status === "DONE") {
+      setStatus("PREPARED");
+    }
```

Comment:
Same as before: create the function in the ApiClient to be called for updating the status (even just a console log is ok), so we only have to update the API, not your components